Makefile

```diff
@@ -40,6 +40,7 @@ doc: .prereq
 
 external:
 	$(MAKE) -C external/libcrt
+	$(MAKE) -C external/openssl -j$$(nproc)
```

Comment:
I think the all target should build all the external stuff, too. That means it should do prereq_external and external. And those should only re-built if needed.

Also, how does this get included in the SVSM binary? Once built this should be part of the SVSM binary so that anything can begin using it.

Reply:
@tlendacky I improved that part in the Makefile.